### DISCUSSION TITLE
Settings

### DISCUSSION THREAD

**Discussion by Thomas282-coder** [ScratchAddons/ScratchAddons#1104] [Help] [answered]
How can I change settings, pls give me the settings link

**Comment by Marc92020** [comment]
If you are talking about scratch addons, you need to have the extension and then go [chrome-extension://fbeffbjdlemaoicjdapfpikkikjoneco/webpages/settings/index.html](url). but if you use another browser than I don't know. I gave the chrome link.

**Comment by GrahamSH-LLK** [comment, marked as answer]
Go to https://scratch.mit.edu/scratch-addons-extension/settings.
